content/docs/user-guide/testing/quality/_index.md

```diff
@@ -0,0 +1,216 @@
+---
+description: Use the Quality system to define quality levels and the console variable settings for each level.
+title: Quality System
+linkTitle: Quality System
+---
```

Comment:
This document seems like it should either be under the existing "visualization" directory or added to a new "quality" or "performance" directory.

I wouldn't consider this directly related to testing.
The impact is more on having the quality level set for the end user to get optimal performance on their device.

Reply:
I'll go with a new section  👍